apps/system/lockscreen/js/source/setting_source.js

```diff
@@ -0,0 +1,52 @@
+/* global SourceEvent */
+'use strict';
+
+/**
+ * Event source for Stream. One Stream can collect events from multiple
+ * sources, which pass different native events (not only DOM events)
+ * to Stream.
+ **/
+(function(exports) {
+  var SettingSource = function(configs) {
```

Comment:
Why? People wouldn't have other `Setting-` prefix events to get confused.
Or we should have a convention to have all sources this naming rule, like 'MinuteClockTickSource'